.travis.yml

```diff
@@ -3,12 +3,12 @@ dist: trusty
 language: php
 
 php:
-  - 5.4
```

Comment:
If we  drop `php-5.4` test, I think the `requirement` block in`composer.json`  should change into `"php": ">=5.5"`.

I'm not sure that this package requires minimum PHP version. Perhaps this will be up to the maintainers' decision.